doc/main.html

```diff
@@ -46,9 +46,9 @@
       <li><cite id="bib-smpte-standards-operation-manual">SMPTE Standards Operation Manual</cite>, v3.1
         <a>https://f.hubspotusercontent00.net/hubfs/5253154/SMPTE%20Standards%20OM%20v3.1.pdf</a></li>
       <li><cite id="bib-smpte-ag-02">SMPTE AG-02</cite>, Document Naming and Packaging
-        <a>https://smpte.github.io/ag-02/</a></li>
+        <a>https://f.hubspotusercontent00.net/hubfs/5253154/Standards%20Files/AG-02-Document-Naming-and-Packaging.pdf</a></li>
```

Comment:
This should point to the governance page... until SMPTE HQ creates permalinks.

Reply:
I agree, will update the other AG references separately.